README.md

```diff
@@ -37,50 +44,83 @@ expBackoffBase = 2
+You can extend and override that file. 
+
+Create a file `java.security` with the following content: 
 
-Required:
-* Linux (the example was tested in Linux Mint 18.3)
-* Java 8 
-* A running Spire server and agent. [HowTo](https://github.com/spiffe/spire#installing-spire-server-and-agent)
-* An entry in the registry that matches this application:
 ```
-cmd/spire-server/spire-server entry create \
--spiffeID spiffe://example.org/workload \
--parentID spiffe://example.org/host \
--selector unix:uid:1000
+# Adding provider following the numeration of the List of Providers in the master file
+security.provider.10=spiffe.provider.SpiffeProvider
+
+# Determines the default key and trust manager factory algorithms for
+# the javax.net.ssl package.
+ssl.KeyManagerFactory.algorithm=Spiffe
+ssl.TrustManagerFactory.algorithm=Spiffe
+
+# The spiffeID that will be trusted
+ssl.spiffe.accept=spiffe://example.org/front-end
 ```
 
-To start the Java server you can run `./gradlew runExample`
+Replace the value of `ssl.spiffe.accept` with the Spiffe ID of the workload that is allowed to connect to your workload.
 
+Pass your custom security properties through the command line via system property: 
+
+```
+-Djava.security.properties=<path to java.security>
 ```
-$ ./gradlew runExample
 
-> Task :run
-[main] INFO spiffe.api.svid.examples.ClientExample - Fetching the SVIDs asynchronously
-[main] INFO spiffe.api.svid.examples.ClientExample - Waiting for certificates...
-[main] INFO spiffe.api.svid.examples.ClientExample - Doing other work...
-[main] INFO spiffe.api.svid.examples.ClientExample - Doing other work...
-[main] INFO spiffe.api.svid.examples.ClientExample - Doing other work...
-[grpc-default-executor-0] INFO spiffe.api.svid.examples.ClientExample - Spiffe ID fetched: spiffe://example.org/workload
-[main] INFO spiffe.api.svid.examples.ClientExample - Exiting...
+The properties defined in your custom properties file will override the properties in the master file. 
+
+### Configure Workload API Socket Endpoint
 
-BUILD SUCCESSFUL in 1s
+The socket endpoint can be configured defining an environment variable named `SPIFFE_ENDPOINT_SOCKET`: 
 
 ```
+export SPIFFE_ENDPOINT_SOCKET=/tmp/agent.sock
+``` 
 
-### Generating the JAR  
+or it can be configured from the command line via system property: 
 
 ```
-./gradlew build
+-Dspiffe.endpoint.socket=/tmp/agent.sock
 ```
 
-The jar file `java-spiffe-0.1-SNAPSHOT.jar` is generated in folder `libs`.
+### Configure a Tomcat connector
 
-### Generating the fat JAR with all the dependencies
+A Tomcat TLS connector that uses the `Spiffe` KeyStore can be configured as follows: 
 
 ```
-./gradlew shadowJar
+<Connector
+            protocol="org.apache.coyote.http11.Http11NioProtocol"
+            port="8443" maxThreads="200"
+            scheme="https" secure="true" SSLEnabled="true"
+            keystoreFile="" keystorePass=""
```

Comment:
those properties are required to be present when configuring a connector with SSL, and they are blank since the Spiffe keystore don't use a file to store the certificates.